dashboard/test/ui/features/step_definitions/section_management_steps.rb

```diff
@@ -103,7 +103,8 @@
   section = JSON.parse(browser_request(url: '/dashboardapi/sections', method: 'POST', body: {login_type: 'email', participant_type: 'student'}))
   section_code = section['code']
   @section_url = "http://studio.code.org/join/#{section_code}"
-  create_user(name, url: "/join/#{section_code}", code: 200, age: under_13 ? '10' : '16')
+  @student = create_user(name, url: "/join/#{section_code}", code: 200, age: under_13 ? '10' : '16')
+  @student
```

Comment:
@Nokondi I thought the `puts` statement was needed to increase our ability to debug the problem in the future. Is there something that has changed since we discussed that?

also, one nit: there's no need to use an `@` variable if you are only referencing it locally. so this can be:
```suggestion
  student = create_user(name, url: "/join/#{section_code}", code: 200, age: under_13 ? '10' : '16')
  puts student
```

Reply:
RE: our conversation on slack, I've added a new step to account_steps that retrieves that current user's information and displays it in the cucumber logs for debugging.